pyproject.toml

```diff
@@ -4,3 +4,6 @@ omit = ["*/tests/*", "*pygmt/__init__.py"]
 [tool.pytest.ini_options]
 minversion = "6.0"
 addopts = "--verbose --doctest-modules --mpl --mpl-results-path=results"
+
+[tool.isort]
+profile = "black"
```

Comment:
I've used `isort` and quite like it, but I've found different `isort` versions to apply the sort styles rather inconsistently (specifically the standard lib/firstparty/thirdparty sections, see https://pycqa.github.io/isort/#custom-sections-and-ordering). Does this "black" profile keep things simple?

Reply:
> Does this "black" profile keep things simple?

No, `profile = "black"` is equivalent to the following settings:
```
multi_line_output: 3
include_trailing_comma: True
force_grid_wrap: 0
use_parentheses: True
ensure_newline_before_comments: True
line_length: 88
```

As for section orders, the default order (https://pycqa.github.io/isort/docs/configuration/options/#sections) is:
```
('FUTURE', 'STDLIB', 'THIRDPARTY', 'FIRSTPARTY', 'LOCALFOLDER')
```
Do you mean the default values change in different `isort` versions?

Reply:
I think there was a point where they separated well known third-party libaries (e.g. numpy and pandas) from other third-party libraries. Maybe that has changed now with newer `isort` versions, but I'm not too sure.

Do you think we should add other configuration options? I'm looking at xarray's at https://github.com/pydata/xarray/pull/4518 and they have some extra ones besides `black`.

Reply:
OK, I will look at these options later.

Reply:
Most of our examples import some of the four packages: pygmt, numpy, pandas and xarray.

Currently, `isort` sorts them like 
```
import numpy as np
import pandas as pd
import xarray as xr

import pygmt
```
because numpy, pandas, and xarray are known *third-party* packages, and pygmt is *first-party* package (not sure if I understand it correctly).

I don't think I like it. We can add options:
```
known_third_party = "pygmt"
force_to_top = "pygmt
```
so that they will be sorted like:
```
import pygmt
import numpy as np
import pandas as pd
import xarray as xr
```

Reply:
After reading through all the `isort` options, I added the following ones to `pyproject.toml`, similar to https://github.com/pydata/xarray/pull/4518:
```
skip_gitignore = true
force_to_top = "pygmt"
known_third_party = "pygmt"
```
the only exception is `default_section = THIRDPARTY`, as I don't understand what it is, and adding it doesn't change the codes at all.

Reply:
I have to remove `force_to_top`, otherwise pylint will complain.